docs/release/release_notes_v2_0.md

```diff
@@ -1,4 +1,136 @@
 # DAOS Version 2.0 Release Notes
 
-DAOS 2.0 is still in validation and has not been released yet.
-The release is planned for November 2021.
+We are pleased to announce the release of DAOS version 2.0.
+
+
+!!!note
+	The DAOS version 2.0 java/hadoop DAOS connector has been updated to use Log4j version 2.16 and may not include the latest functional and security updates. DAOS 2.0.1 is targeted to be released in January 2022 and will include additional functional and/or security updates. Customers should update to the latest version as it becomes available.
+
```

Comment:
please add line breaks to keep lines <80char wide. otherwise, comparing future diffs will be nunecessarily hard.